src/prefabs/dataTable.js

```diff
@@ -401,6 +401,12 @@
             },
           },
         },
+        {
+          type: 'TOGGLE',
+          label: 'Enable first/last page navigation',
+          key: 'enableFirstLastPageNavigation',
+          value: false,
+        },
```

Comment:
make sure to include this option to all the prefabs where a dataTable component is being used :pray: